src/virtualenv/app_data/via_disk_folder.py

```diff
@@ -154,10 +154,7 @@ def locked(self):
 
     def write(self, content):
         folder = self.file.parent
-        try:
-            folder.mkdir(parents=True, exist_ok=True)
-        except OSError:
-            pass
+        folder.mkdir(parents=True, exist_ok=True)
```

Comment:
since we're using pathlib `exist_ok` -- an `OSError` here means there's a real failure